target_worker/target_worker.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def clustering():
```

Comment:
Both these functions doesn't need to be functions at all. It can be a simple dict or tuple in the init file - with conditional content, based on the environ. :wink:  That would be even nice.

Reply:
I have removed `target_worker/target_worker.py` altogether

All the necessary stuff is in `target_worker/__init__.py` -- good suggestion!